trueos-handbook/README.md

```diff
@@ -102,6 +102,11 @@ using TrueOS and/or FreeBSD before following these instructions.
 
 ### Required Applications <a name="reaps"></a>
 
+.. note:: In case you are building the documentation from an OS other that
+   TrueOS, please note that the only supported `make` tool is (obviously)
+   the one provided with TrueOS (NetBSD make). So, please make sure you are
+   using the correct binary (for example, `bmake` for Debian-based systems). 
```

Comment:
> (for example, `bmake` for Debian-based systems).

This aside is fine as its own sentence:

"... correct binary. For example, Debian-based systems use `bmake`."